nip47/controllers/get_info_controller.go

```diff
@@ -70,6 +72,27 @@ func (controller *nip47Controller) HandleGetInfoEvent(ctx context.Context, nip47
 		responsePayload.Network = network
 		responsePayload.BlockHeight = info.BlockHeight
 		responsePayload.BlockHash = info.BlockHash
+
+		if app != nil {
+			metadata := map[string]interface{}{}
+			if app.Metadata != nil {
+				jsonErr := json.Unmarshal(app.Metadata, &metadata)
```

Comment:
this means we also return the metadata that the apps are setting?

Reply:
Yes, do you see any issues with this?